src/main/java/com/erigitic/commands/JobCommand.java

```diff
@@ -219,23 +224,61 @@ public CommandResult execute(CommandSource src, CommandContext args) throws Comm
 
                 if (optSet.isPresent()) {
                     TEJobSet jobSet = optSet.get();
-                    Currency defaultCurrency = totalEconomy.getDefaultCurrency();
 
-                    for (TEActionReward actionReward : jobSet.getActionRewards()) {
-                        lines.add(Text.of(TextColors.GOLD, "[", jobManager.titleize(actionReward.getAction()), "] ", TextColors.GRAY, actionReward.getTargetID(), TextColors.GOLD, " (", actionReward.getExpReward(), " EXP) (", defaultCurrency.format(actionReward.getMoneyReward()), ")"));
+                    for (TEAction action : jobSet.getActions()) {
+                        Text listText;
+
+                        if (action.isIdTraited()) {
+                            // MC does not support '\t'
+                            String tab = new String(new char[action.getAction().length() + 2]).replace("\0", " ");
+                            List<Text> texts = new ArrayList<>(action.getRewards().size());
+
+                            action.getRewards().forEach((k, v) -> {
+                                Text metaText = Text.of("");
+
+                                if (action.isGrowing() && extended) {
+                                    metaText = Text.of(",growing=1");
+                                }
+
+                                Text rewardText = (Text.of('\n', tab,  TextColors.GRAY, "{", action.getIdTrait(), '=', k, metaText, "} ", TextColors.GOLD, formatReward(v)));
```

Comment:
I'm not exactly sure what you mean by another tab, can you explain further?

Reply:
I think it would look better if the second line of information where it displays the IdTrait, should be tabbed over one more. For example:

techreborn:techreborn.ore (10 exp) ($10.00)
    {type=bauxite} <-- This part should be tabbed over one more

Reply:
Use double quotes instead of single quotes